### DISCUSSION TITLE
Input field will not accept the space key

### DISCUSSION THREAD

**Discussion by drdan18** [livewire/livewire#7447] [1. Help]
A very odd behaviour.  

I have setup a livewire component with a form using a form object.   

When I use the form to create a new record it works fine. 

When I use the form to edit a record it won't allow a space in the input field.  All other keys work fine.   The code for the form in use is a copy and paste.  

Any obvious ideas that would stop the recognition of a space key?

**Comment by joshhanley** [comment]
@drdan18 please share your code so we can have a look 🙂

**Reply by drdan18** [reply]
I moved the Edit Form outside of the x-menu and it works as expected.  I must have done something wrong with the x-menu component.

**Comment by SaedyMoz** [comment]
I faced a similar problem. livewire screencast tutorials ; Modals episode 8- adding a dropdown menu , using x-menu. also selecting texts on dialog by mouse not working. other keys : Ctrl+c , ...

**Reply by SaedyMoz** [reply]
using teleportation in next episode(9-teleporting) solve the problem.

**Reply by drdan18** [reply]
Thanks.  I was thinking teleportation might help.   I have abandoned x-menu and x-dialog for now because of all the issues.  Another main one was a very noticeable flicker of then entire model when typing in an input field.   Also if you held down the backspace key it would crash the input field clearing and not allowing further input.  

I will check out teleportation.
